mypy/nodes.py

```diff
@@ -370,13 +370,20 @@ def __str__(self) -> str:
         return 'ImportedName(%s)' % self.target_fullname
 
 
+FUNCBASE_FLAGS = [
+    'is_property', 'is_class', 'is_static',
+]
```

Comment:
Shouldn't we also update `astdiff.py` according to these flag reshuffling? This may break fine grained incremental mode (in some corner cases).

Reply:
Ooh, good point -- I didn't even know that file existed.

I tried making the change + tried adding an test to one of the fine-grained incremental tests. (I'm pretty unfamiliar with fine-grained incremental stuff though, so let me know if I did it incorrectly.)